05.ls/ls1.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# !/usr/bin/env ruby
+NUMBEROFCOLUMNS = 3
```

Comment:
単語の切れ目はアンダースコアにしましょう。
参考：https://www.javadrive.jp/ruby/const/index2.html

あとは、一行空行を開けた方が見やすくなります。（ここまでは定数の宣言だよ！ここからはプログラムが始まるよ！みたいな感じで論理的な塊と塊の間には空行を置きたいところです）

```suggestion
NUMBER_OF_COLUMNS = 3

```